dal/database_connection.py

```diff
@@ -0,0 +1,10 @@
+from beanie import init_beanie
+from models.user import UserModel
+from config import config
+
+async def connect_to_db():
+    await init_beanie(
+        database=config['MONGO_URI'],
+        document_models=[UserModel],
+        uri=config['MONGO_URI']
+    )
```

Comment:
make it 1 line

Reply:
done

Reply:
Version numbers are usually considered metadata about the application itself rather than a configuration option. They are typically defined separately, either directly in the codebase or in a dedicated file specifically for metadata like version.py or metadata.py. Keeping version information separate from configuration settings helps maintain a clear distinction between configuration and metadata, making the codebase more organized and easier to manage.

`# metadata.py

__version__ = "1.0.0"
`

Reply:
I created a new file and called it version.py.